package/openmrs-config/configuration/concepts/questions.csv

```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Fully specified name:es,Short name:en,Short name:fr,Short name:es,Description:en,Description:fr,Description:es,Data class,Data type,_version:1,_order:1000
```

Comment:
Could you rename the file to be **patient_notes_concepts.csv** for now?
We shall see later if we rename again.

Reply:
The thought was to group by "Data Class" and so it would not only have patient_notes concept but any other question concept in a generic bare minimum scenario.